docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -157,6 +157,18 @@ Property                                              Description
                                                       from BigQuery.
                                                       Please read this `section <#arrow-serialization-support>`_
                                                       before enabling this feature.
+``bigquery.rpc-proxy.uri``                            Proxy URI to use if connecting through a proxy.
+``bigquery.rpc-proxy.username``                       Proxy user name to use if connecting through a proxy.
+``bigquery.rpc-proxy.password``                       Proxy password to use if connecting through a proxy.
+``bigquery.rpc-proxy.key-store-path``                 Keystore containing client certificates to present to proxy if
+                                                      connecting through a proxy. Only required if proxy uses
+                                                      mutual TLS.
+``bigquery.rpc-proxy.key-store-password``             Password of the keystore specified by
+                                                      ``bigquery.rpc-proxy.key-store-path``.
+``bigquery.rpc-proxy.trust-store-path``               Truststore containing certificates of the proxy server if
+                                                      connecting through a proxy.
+``bigquery.rpc-proxy.trust-store-password``           Password of the truststore specified by
+                                                      ``bigquery.rpc-proxy.trust-store-path``.
```

Comment:
Most connectors use trutstore- * & keystore-* for TLS config property names. Why did we use trust-store & key-store here?

Reply:
those are used for TLS - since BigQuery APIs are exposed over HTTPS the proxy needs to be connected via TLS and it's certificate needs to be added to truststore so that the HTTPS connection is trusted.

You cannot proxy BigQuery APIs without TLS.

Reply:
Sorry, I wanted to ask if we can remove `-` from properties like `bigquery.rpc-proxy.keystore-path`
```
➜  git grep -e '@Config' --and -e 'truststore' | wc -l
      18
➜  git grep -e '@Config' --and -e 'trust-store' | wc -l
       2

➜  git grep -e '@Config' --and -e 'keystore' | wc -l
      25
➜  git grep -e '@Config' --and -e 'key-store' | wc -l
       2
```